awscli/customizations/s3/subcommands.py

```diff
@@ -593,7 +598,10 @@ class MbCommand(S3TransferCommand):
 
 class RbCommand(S3TransferCommand):
     NAME = 'rb'
-    DESCRIPTION = "Deletes an S3 bucket."
+    DESCRIPTION = (
+        "Deletes an empty S3 bucket. A bucket must be completely empty "
+        "of objects and versioned objects before it can be deleted."
```

Comment:
This makes is seem like you can't use `rb` if you have non-versioned objects in your bucket.  Should this be reworded?  Or maybe `--force` should be mentioned here.